include/spdlog/details/os.h

```diff
@@ -32,11 +32,16 @@ SPDLOG_API std::tm gmtime() SPDLOG_NOEXCEPT;
 SPDLOG_CONSTEXPR static const char *default_eol = SPDLOG_EOL;
 
 // folder separator
+#if !defined(SPDLOG_FOLDER_SEPS)
 #ifdef _WIN32
-static const char folder_sep = '\\';
+#define SPDLOG_FOLDER_SEPS "\\/"
 #else
-SPDLOG_CONSTEXPR static const char folder_sep = '/';
+#define SPDLOG_FOLDER_SEPS "/"
```

Comment:
Seems incorrect. What would happen if a folder name in windows contains the "\" character ? Let's keep the orig code

Reply:
Folders can't contain this character.